cmd/thanos/compact.go

```diff
@@ -247,7 +247,8 @@ func runCompact(
 		prober.NewInstrumentation(component, logger, extprom.WrapRegistererWithPrefix("thanos_", reg)),
 	)
 
-	srv := httpserver.New(logger, reg, component, httpProbe,
+	srv := httpserver.New(logger, reg, component, "/",
```

Comment:
Why not we route as well?

Reply:
I checked existing compacter [flags](https://thanos.io/components/compact.md/#flags) in which there was `web.route-prefix`. I was not sure whether I should add `web.route-prefix` flag to all components.

Reply:
Should I add `web.route-prefix` to all components?

Reply:
ping

Reply:
May be we can do it in a separate PR.

Reply:
I think it would be nice to have `--web.route-prefix` in other components as well, but let's leave it for a separate PR.